GC.md

```diff
@@ -0,0 +1,207 @@
+NIP-GC
+======
+
+Geocaching Events
+-----------------
+
+`draft` `optional`
+
+This NIP defines event kinds for geocaching activities on Nostr, enabling users to create, discover, and log geocaches in a decentralized manner.
+
+## Motivation
+
+Geocaching is a global treasure hunting game where participants use GPS coordinates to hide and seek containers ("geocaches") at specific locations. This NIP brings geocaching to Nostr, allowing for censorship-resistant, decentralized geocache listings and logs without relying on centralized platforms.
+
+## Event Kinds
+
+This NIP defines two new event kinds:
+
+- `kind:37515`: Geocache listing (addressable event)
+- `kind:37516`: Geocache log entry
+
+## Geocache Listing (kind:37515)
+
+A geocache listing is an addressable event that describes a hidden geocache. The event is addressable to allow cache owners to update their listings (e.g., to provide maintenance updates or adjust coordinates).
+
+### Content
+
+The `.content` field contains the cache description as plain text.
+
+### Tags
+
+Required tags:
+- `d` (required): Unique identifier for the geocache
```

Comment:
this doesn't seem to be necessary, each geocache is unique so their identifier is naturally their event id

Reply:
```suggestion
- `d` (required): Unique identifier for the geocache
```